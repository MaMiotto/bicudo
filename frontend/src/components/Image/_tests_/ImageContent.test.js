import React from 'react';
import { render, screen } from "@testing-library/react";
import { shallow } from 'enzyme';
import ImageContent from '../';
import Worker from '../../../images/worker.jpg';

test('it should render an image', () => {
    const logo = shallow(<ImageContent src={Worker}/>);
    expect(logo.find({prop:'src'})).toBeTruthy();
});

test('it should match snapshot', () => {
    const { container } = render(<ImageContent />);
    expect(container.firstChild).toMatchSnapshot();
});

test('should have src attribute', () => {
    render(<ImageContent image={Worker}/>);
    const logo = screen.getByRole('img');
    expect(logo).toHaveAttribute('src',Worker);
})


