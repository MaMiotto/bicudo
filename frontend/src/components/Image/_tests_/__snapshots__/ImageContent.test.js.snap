// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match snapshot 1`] = `
<div
  class="sc-bdnxRM wNuqL"
>
  <img
    class="sc-gtsrHT gqaJrf"
  />
</div>
`;
